tests/sys/vane/ford.hoon

```diff
@@ -788,14 +788,14 @@
       ::  when we scry on a blocked path, expect a subscription move
       ::
       ^=  call-args
-        :*  duct=~[/one]  type=~  %build  ~nul  live=%.n
+        :*  duct=~[/one]  type=~  %build  live=%.n
             [%scry %c ren=%x rail=[[~nul %desk] /bar/foo]]
         ==
       ::
       ^=  moves
         :~  :*  duct=~[/one]  %pass
                 wire=/~nul/scry-request/cx/~nul/desk/~1234.5.6/foo/bar
```

Comment:
At some point we should pull `our` out of the wires that Ford uses, too. I can do it in a later PR if you'd like.